docker-compose.yml

```diff
@@ -51,10 +40,17 @@ services:
     networks:
       - dex-network
 
+  next-app:
+    build:
+      context: ./next-app
+      dockerfile: Dockerfile
+    ports:
+      - "3001:3000"
```

Comment:
Why in your docker-cpmpose u define a port redirection from 3000 to 3001 for exposing it at 3000 in your dockerfile?

Reply:
My bad

Reply:
Actually no. That was write but I modified it anyway.

EXTERNAL:INTERNAL